infoschema/tables_test.go

```diff
@@ -86,14 +86,22 @@ type testClusterTableSuite struct {
 
 func (s *testClusterTableSuite) SetUpSuite(c *C) {
 	s.testTableSuite.SetUpSuite(c)
-	s.rpcserver = setUpRPCService(c, "0.0.0.0:10080")
+	s.rpcserver = s.setUpRPCService(c, "0.0.0.0:10080", s.dom)
 	s.httpServer, s.mockAddr = setUpMockPDHTTPSercer()
 }
 
-func setUpRPCService(c *C, addr string) *grpc.Server {
+func (s *testClusterTableSuite) setUpRPCService(c *C, addr string, dom *domain.Domain) *grpc.Server {
```

Comment:
The `domain.Domain` can be access by `s` and we don't need to use the parameter to pass it.

Reply:
done.